X/x265/build_tarballs.jl

```diff
@@ -3,12 +3,16 @@
 using BinaryBuilder
 
 name = "x265"
-version = v"3.6"
+version = v"4.0"
+
+# NOTE: The release notes for version 4.0 do not mention any
+# incompatibility with version 3.6. Packages currently using 3.6 might
+# try building against 4.0.
```

Comment:
The ABI was broken: https://github.com/JuliaBinaryWrappers/x265_jll.jl/commit/63d80b3d42ec2e4d16e523462866b3de9f75a17f